### PR TITLE
man: Use ukify instead of objcopy in examples

### PR DESCRIPTION
**Pull request by medhefgo**
These hardcoded VMA section offsets are a terrible thing and should
vanish from earth.

### CODE REVIEW

File: man/systemd-stub.xml [hunk 1]
```diff
@@ -432,8 +395,7 @@
       <citerefentry><refentrytitle>systemd-sysext</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
       <ulink url="https://uapi-group.org/specifications/specs/boot_loader_specification">Boot Loader Specification</ulink>,
       <ulink url="https://systemd.io/BOOT_LOADER_INTERFACE">Boot Loader Interface</ulink>,
-      <citerefentry project='man-pages'><refentrytitle>objcopy</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
-      <citerefentry project='archlinux'><refentrytitle>sbsign</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
+      <citerefentry><refentrytitle>ukify</refentrytitle><manvolnum>1</manvolnum></citerefentry>
```
Comment (keszybz) [inline man/systemd-stub.xml]: Comma missing at the end.

**Review by keszybz — COMMENTED** [review]
LGTM.

> From looking at the docs, it would probably be helpful for ukify to learn an arg to spit out the json from measure to a file.

Yeah, we can certainly do that.

> It also seems to be lacking the --append arg for measuring?

It implements `--apend` internally. You specify multiple keys and/or multiple phase paths and they get signed and merged. But maybe I'm misunderstanding what you want to do… It's probably best to file an issue so we can discuss the usecase.

**Comment by medhefgo** [comment]
> It implements `--apend` internally. You specify multiple keys and/or multiple phase paths and they get signed and merged. But maybe I'm misunderstanding what you want to do… It's probably best to file an issue so we can discuss the usecase.

I was simply looking at the `Introduce a second public key, signing the same kernel PCR measurements, but only for the initrd boot phase` example of systemd-measure. It does two sd-measure signs and then a objcopy. Without me really having any idea how all this stuff works I only replaced the objcopy with ukify by manually specifying the pcr sections from the measure output. I *think* that all three (or just the last two?) commands are replaceable by ukify, I just dunno quite how and if that would require some `--append` support or not.

**Comment by keszybz** [comment]
Oh, I see now. Let's merge this as-is, because it is correct. I'll submit a separate patch to reduce the number of steps.

**Comment by keszybz** [comment]
Screenshot from 2023-01-12 09-07-33

This doesn't look right. @DaanDeMeyer you touched the rules the last time… Maybe it sets the label on any change to the PR now?

**Comment by keszybz** [comment]
→ https://github.com/systemd/systemd/pull/26029

**Pull request merged**